src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -226,11 +226,11 @@ Just set the `pinned` prop.
 					class="app-navigation-entry-link"
 					:aria-description="ariaDescription"
 					:aria-expanded="opened.toString()"
-					:href="href || '#'"
+					:href="href || routerLinkHref || '#'"
 					:target="isExternal(href) ? '_blank' : ''"
 					:title="title || nameTitleFallback"
 					@blur="handleBlur"
-					@click="(event) => onClick(event, navigate)"
+					@click="(event) => onClick(event, navigate, routerLinkHref)"
```

Comment:
Vue wraps and hoists handler in `v-on` directives. Otherwise it creates handler on each render, if I'm not mistaken.

```suggestion
					@click="onClick($event, navigate, routerLinkHref)"
```